aemedge/blocks/contact/contact.css

```diff
@@ -2,80 +2,82 @@
   padding-top: var(--udexSpacer24);
 }
 
-.contact-wrapper .logo-lang-picker {
-  display: flex;
-  flex-direction: row;
-  padding-bottom: var(--udexSpacer48);
-}
-
-.contact-wrapper .logo-lang-picker select {
-  padding: var(--udexSpacer4) var(--udexSpacer12);
-  border-radius: 4px;
-  border: 1px;
-  min-width: 282px;
-  font-size: var(--udexTypographyBodyXSFontSize);
-  letter-spacing: 0;
-  text-align: left;
-  color: var(--udexColorGrey7);
-  background-color: var(--udexColorNeutralWhite);
-}
-
-.contact-wrapper .icon-sap-logo {
-  height: 32px;
-  width: 65.4px;
-  padding-right: var(--udexSpacer20);
+.contact-wrapper {
+  .logo-lang-picker {
```

Comment:
Do we have any policies regarding minimum support for css features? This feature seems to be supported in browsers with ~80% of the users (https://caniuse.com/css-nesting), is this enough for this site? /cc @saurabh-khare @mhaack